lib/metrics.js

```diff
@@ -54,6 +55,14 @@ const ENV_VAR_METRICS = {
     }
 };
 
+function truncateString(str = "", maxLength = DEFAULT_MAX_STRING_LENGTH) {
+    if (str.length > maxLength) {
+        return str.substring(0, maxLength - 3) + "...";
+    } else {
```

Comment:
I get why the else is here, but it's not needed, since the else calse would execute the last line.

Reply:
Yes, it's for the humans not the language runtime. :P  Your way is faster in some some languages which convert else statements to "Goto label" statements at the end of the day, but I don't think compile time performance is a major factor here so it's a style over function, so I went for block-level symmetry.  If that's not ok the I could just write it out as a unary expression too.